docs/pages/enterprise/workflow/resource-requests.mdx

```diff
@@ -26,6 +26,13 @@ available in Teleport Enterprise.
 
 (!docs/pages/includes/tctl.mdx!)
 
+<Admonition type="warning">
+All Teleport Nodes in the cluster must be running Teleport `v10.0.0` or greater
+in order for Resource Access Requests to be properly enforced.
+Nodes on older versions will only do RBAC checks based on roles and may not
```

Comment:
Are there any conditions we can list for when Nodes may not respect resource restrictions? Any conditions for when they _would_ respect resource restrictions?

Reply:
They never respect resource restrictions when running versions < 10, I used "may" because access may be blocked anyways by the roles. It's possible to securely set up a cluster with resource access requests and some nodes on v9 if none of the older nodes can be accessed by any of the `search_as_roles`. I wasn't sure if I should elaborate on all that or it would just muddy the waters, easier to just say everything must be on v10

Reply:
I think some more detail here would be great so readers understand when different Teleport services will/won't respect resource restrictions on older versions. If there isn't a comprehensive set of conditions we can list here, then I think a quick example will be okay!

Reply:
This is basically just a warning that resource access requests are a Teleport 10 feature and they will not be consistently enforced if your cluster is running a mix of Teleport 9 and Teleport 10 software. You need to fully upgrade to 10.

Reply:
@ptgott It's not supported at all to have any instances on versions older than v10, I don't think it's worth elaborating on specific workarounds to maybe make that work, I shouldn't have mentioned that in my comment here. I'm trying to say "Do not use this before upgrading everything to v10, if anything is on v9 this feature is fundamentally broken and insecure". I updated the wording a bit to try to make that more clear.